allennlp/experiments/driver.py

```diff
@@ -0,0 +1,23 @@
+from allennlp.common import Params
```

Comment:
I know I initially set this up, but now that I see it like this, I think it's too much.  I don't think we need this complexity at the top level, and it's a confusing interface to expose to users of the library.  I think we should just stick with a `Trainer` and an `Evaluator`, or something, and forget about the notion of `Drivers` - just have a script that runs training, and a script that runs evaluation (or two different flags to the same script), like how we had it in DeepQA.

Luckily, I don't think that requires very much change in the code - just remove this class, rename `TrainDriver` and remove the inheritance, and change `execute_driver.py` to `run.py`, like in DeepQA.